### DISCUSSION TITLE
Tests T1

### DISCUSSION THREAD

**Discussion by bruverie** [IIC2133-PUC/2024-2#92] [Tarea 1] [answered]
Tengo algunos problemas con el test medium de la parte 1:

1. En el output, se espera un texto para el comando FESTIVAL distinto al que se solicita en el enunciado: Place en vez de Sector y sin los ":".
2. En la línea 19 del output se espera esto:
Medians resume:
	3 v/s 0
	82 - 46
y utilizando mensajes para debugguear mi código estoy obteniendo lo siguiente:
Debugging Medians:
Sector 0: Median = 46
Sector 1: Median = 82
Sector 2: Median = 72
Sector 3: Median = 82
Sector 4: Median = 55
Final Medians:
Best Median = 82 (Sector 1)
Worst Median = 46 (Sector 0)
Medians resume:
	1 v/s 0
	82 - 46
Según lo que entiendo del enunciado, el empate en medianas debería dejar como best median al sector 1 por tener un ID mas bajo o estoy mal en eso?

**Comment by 4n1t4** [comment, marked as answer]
hola! revisando el generador y las pautas tenesmo que no deberías tener este empate. De forma específica , tu bug debe estar relacionado a la forma en la que manejas la actualización de la mediana. Verás, en el input en cuestión tenemos que en relación al sector 1 ocurren los siguientes eventos que lo afectan  antes de pedir la median con la cual tienes problemas. 

En específico :

ENTER 1 3 34
ENTER 1 4 82
ENTER 1 5 96
.....
ENTER 1 15 20
.....
ENTER 1 19 17
...
LEAVE ( se va el 19 del sec 1 ) 
(luego de esto viene medians que indicas)

Como verás para la median que te piden el valor del sector 1 deberia haber quedado en 58 = (34+82)//2.

**Reply by bruverie** [reply]
Toda la razón! muchas gracias.

Respecto al primer punto, me guío por el enunciado o por el output esperado?

**Reply by 4n1t4** [reply]
El output esperado corregirlos el enunciado para un caso de desempate

**Reply by Segismundios** [reply]
Holaa @bruverie. Guíate por el output esperado, pronto se actualizará el enunciado.
Saludos!
